src/scale_audio_rating.py

```diff
@@ -264,6 +279,10 @@ async def main(cfg, args):
     if not os.path.exists(output_dir):
         os.mkdir(output_dir)
 
+    # create batch
+    if args.create_batch:
+        batch = await create_batch(cfg.get("CommonAccountKeys", 'ScaleAPIKey'),args.project,args.clips,)

```

Comment:
what's the comma at the end?
```suggestion
        batch = await create_batch(cfg.get("CommonAccountKeys", 'ScaleAPIKey'),cfg.get("CommonAccountKeys", 'ScaleAccountName'),args.clips)

```